release-notes/opensearch-dashboards.release-notes-1.3.8.md

```diff
@@ -0,0 +1,32 @@
+# Version 1.3.8 Release Notes
+
+
+### 🛡 Security
+
+- [CVE-2022-25901] Bump supertest ([#3326](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3326))
+- [CVE-2022-25860] Bumps simple-git from 3.15.1 to 3.16.0 ([#3345](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3345))
+- [CVE-2022-46175] Bumps json5 version from 1.0.1 and 2.2.1 to 1.0.2 and 2.2.3 ([#3201](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3201))
+- [CVE-2022-25912] Bumps simple-git from 3.4.0 to 3.15.0 ([#3036](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3036))
+
+
+### 📈 Features/Enhancements
+
+- [Optimizer] Fixes misleading embeddable plugin error message ([#3043](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3043))
+- [Optimizer] Trim trailing slashes before checking no-restricted-path rule ([#3020](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3020))
+
+
+### 🚞 Infrastructure
+
+- Lock workflow tests to Chrome and ChromeDriver 107 as the last combination that run on Node.js v10 ([#3299](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3299))
```

Comment:
Is #3299 titled `Enhance the Build and Test workflow `? I see #3299 in both Infrastructure and Tests... based on the contents, maybe putting it in one place and titling it as its original tile will cause less confusion?

Reply:
Remove the entry under Test.